zerver/views/message_send.py

```diff
@@ -193,7 +193,7 @@ def handle_deferred_message(
 def send_message_backend(
     request: HttpRequest,
     user_profile: UserProfile,
-    message_type_name: str = REQ("type"),
+    req_type: str = REQ("type", str_validator=check_string_in(Message.API_RECIPIENT_TYPES)),
```

Comment:
I went with the `req_` format we use in a few places here to clarify this is a parameter from the request.

Reply:
That works for me!